src/Framework/FieldsAPI/Actions/UpdateValidationRulesWithOptionalAsDefault.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Give\Framework\FieldsAPI\Actions;
+
+use Give\Vendors\StellarWP\Validation\Rules\ExcludeIf;
+use Give\Vendors\StellarWP\Validation\Rules\ExcludeUnless;
+use Give\Vendors\StellarWP\Validation\Rules\Optional;
+use Give\Vendors\StellarWP\Validation\ValidationRuleSet;
+
+class UpdateValidationRulesWithOptionalAsDefault
+{
+    /**
+     * This adds the "optional" rule to fields that don't have a "required" rule.
+     * This is to ensure that fields that are not required are not validated unless they have a value.
+     * Additionally, this ensures that the "optional" rule is placed before the "exclude" rules to preserve the intended pipeline functionality.
+     *
+     * @unreleased
+     */
+    public function __invoke(ValidationRuleSet $rules): ValidationRuleSet
+    {
+        if (!$rules->hasRules()) {
+            return $rules;
+        }
```

Comment:
Seems like we can also bail out early if `$rules->hasRule('optional')`?

Reply:
I think you're right - this should basically be ignored if someone is manually using optional.

Reply:
Updated here e90bae2c